src/cpp/utils/threading/threading_empty.ipp

```diff
@@ -15,24 +15,42 @@
 namespace eprosima {
 
 void set_name_to_current_thread(
+        char*, /* thread_name_buffer */
         const char* /* name */)
 {
 }
 
 void set_name_to_current_thread(
+        char*, /* thread_name_buffer */
         const char* /* fmt */,
         uint32_t /* arg */)
 {
 }
 
 void set_name_to_current_thread(
+        const char* /* fmt */,
+        uint32_t /* arg */)
+{
+}
+
+void set_name_to_current_thread(
```

Comment:
```suggestion

```

Reply:
We need the proxy function overload with only one `arg` because is being externally called from `TCPTransportInterface.pp:1102`

Reply:
I think we could remove the thread name change there. The name was already set with the current value of the port when the thread was created.